src/styles/ionic.scss

```diff
@@ -47,4 +47,10 @@ ion-toolbar {
   --ripple-color: transparent;
   --background-focused: transparent;
   --background-activated: transparent;
-}
\ No newline at end of file
+}
+
+ion-searchbar {
+  input {
+    font-size: small !important;
```

Comment:
it will be nice to use numbers instead of keywords, to make it more accurate and easy to manipulate